rllib/connectors/tests/test_agent.py

```diff
@@ -401,7 +401,9 @@ def test_vr_connector_with_multiple_buffers(self):
             ),
             # act[t-context_len+1:t]
             "context_act": ViewRequirement(
-                SampleBatch.ACTIONS, shift=f"-{context_len-1}:-1"
+                SampleBatch.ACTIONS,
+                shift=f"-{context_len-1}:-1",
+                space=Box(-np.inf, np.inf, shape=(1,)),
```

Comment:
why did u need to adde space here?

Reply:
Because that's the proper dimension of the action space. I caught that error in my other PR already. It's because of the `fill_value = get_dummy_batch_for_space(
                        space,
                        batch_size=0,
                    )` that is also part of this PR.

Reply:
Is it not the proper dimension? As far as I now understand the agent collector, this should be proper!

Reply:
if you don't provide the space here, what's the failure exactly? From reading this unit-test the reasoning behind the test should be independent of the space.

Reply:
Now also added this to the obs view requirements.
These are all checked now